code/modules/mob/living/carbon/human/death.dm

```diff
@@ -13,9 +13,9 @@
 
 		playsound(src.loc, 'sound/effects/gib.ogg', 100, 1, 10)
 
-	for(var/obj/item/organ/I in internal_organs)
+	for(var/obj/item/organ/internal/I in internal_organs)
 		if(istype(loc,/turf))
-			I.removed()
+			I.remove(src)

```

Comment:
yeah this is just removing the organ from the player organ list, i think, i will double check if it sets it on the ground.. regardless, what should i change it to to throw all te things?